charts/pxc-db/Chart.yaml

```diff
@@ -1,9 +1,9 @@
-apiVersion: v1
-appVersion: "1.1.0"
+apiVersion: v1-3-0
```

Comment:
this change of API version is probably going to cause issues, this is the version for the Helm API.  see https://helm.sh/docs/topics/charts/#the-apiversion-field

Reply:
done